src/Views/CatalogItemInfo.html

```diff
@@ -38,18 +38,32 @@
             <h1 data-bind="text: catalogItem.name"></h1>
         </div>
         <div class="catalog-item-info-content">
+            <!-- ko if: catalogItem.description -->
             <div class="catalog-item-info-section">
-                <!-- ko if: catalogItem.description -->
+                <h2>Description</h2>
                 <div class="catalog-item-info-description" data-bind="sanitizedHtml: catalogItem.description"></div>
-                <!-- /ko -->
-                <!-- ko if: !catalogItem.description -->
+            </div>
+            <!-- /ko -->
+
+            <!-- ko if:
+            !catalogItem.description &&
+            (!catalogItem.info || !catalogItem.info.some(function(i){return (i.name === "Description" || i.name == "description")}))
```

Comment:
It might be good to move this complicated-ish logic to a computed property on the view-model instead of sticking it here in the view.  You can define a computed property with `knockout.defineProperty`.